_rules/inline-link-in-paragraph-has-distinguishable-border-36f116.md

```diff
@@ -0,0 +1,182 @@
+---
+id: 36f116
+name: Inline link has distinguishable border
+rule_type: atomic
+description: |
+  This rule checks that inline links have a border that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: The element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: The element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][].
+
+## Expectation
+
+Each target element has a [visible][] [border](https://drafts.csswg.org/css-backgrounds/#borders).
```

Comment:
What are the cases when this rule passes but "distinguishable style not based on color" doesn't? (if there are none, then this rule is useless).
I fell that would be cases where the border is visible but not distinguishing because there are borders on other elements, but then that would fail the second assumption… 🤔

Reply:
I have to say that I can't fault your reasoning